src/locales/en/concerts.yaml

```diff
@@ -339,4 +339,14 @@ concerts:
       27 enthusiastic students, 3 pieces and 1 conductor! That’s the 2020 edition of D.S.M.G. Krashna Musika’s chamber orchestra, led by Ben Weishaupt. The orchestra will perform a diverse selection of pieces from the baroque, classical, and modern periods: Bach’s first orchestral suite, Haydn’s 45th symphony (“Farewell”) and pieces from Shostakovich’s 24 preludes and fugues. This last work has been jointly arranged for chamber orchestra by four members of Krashna Musika, especially for this concert!
 
       Curious? On Saturday, March 14, Krashna’s chamber orchestra will perform these pieces in the Lutherse Kerk Delft, at 20:15! Tickets are €13 (regular) and €8 (under 25 years old) when bought at the door. Get a 50ct discount per ticket by reserving in advance: Send a mail to [krashko@krashna.nl](mailto:krashko@krashna.nl) with the number of tickets you would like to reserve and your name.
-                                                                                                                                                                                                                               ![Digital Flyer](/images/krashko-2020-digiflyer.png?style=centerme-75)
+
+  livestream-kamu-august-2020:
+    title: Krashna Chamber Music Concert Livestream
+    description: |
+      In recent months, Krashna has had to cancel many events due to the measures surrounding Covid-19. In order to give a nice end to this academic year, the chamber music department of Krashna Musika has organized a very special concert, namely: a chamber music concert livestream, which can be viewed on YouTube.
+      
+      We will start on \bold(Sunday 2 August at 14:30). Musicians will make live music in two locations. It is professionally filmed with the collaboration of the livestream committee of rowing association Proteus-Eretes, so you can listen and watch it from your own home. 
+      
+      The concert will last approximately 45 to 50 minutes. During the concert parts of the Credo in F by Lotti and the first part of Schubert's Rosamunde string quartet will be performed. 
+
+      The link to the YouTube event can be [found here](https://youtu.be/czwuJ2rG0gc).
```

Comment:
Could you add an empty newline to this file, that is the way it is supposed to be :)